voice-client-javascript/assets/admin/statuses.private.js

```diff
@@ -0,0 +1,256 @@
+
+Would you like to use this app?
+`;
+      status.actions = [
+        {
+          title: "Use existing TwiML application",
+          name: "useExistingTwimlApp",
+          params: {
+            twimlApplicationSid: app.sid
+          }
+        },
+        {
+          title: "Do not use existing TwiML application, create a new one",
+          name: "createTwimlApp",
+          params: {
+            friendlyName
+          }
+        }
+      ];
+    } else {
+      status.description = `We need to create a new TwiML Application. You can do this by clicking the button below. 
+          
+You can do this [via the API or CLI](https://www.twilio.com/docs/usage/api/applications?code-sample=code-create-a-new-application-within-your-account&code-language=curl&code-sdk-version=json).`;
+      status.actions = [
+        {
+          title: "Create a new TwiML Application",
+          name: "createTwimlApp",
+          params: {
+            friendlyName
+          }
+        }
+      ];
+    }
+  }
+  return status;
+}
+
+async function getCallerIdStatus(context) {
+  const client = context.getTwilioClient();
+  const callerId = process.env.CALLER_ID;
+  const status = {
+    valid: false,
+    title: "Caller ID is set to a valid number"
+  };
+  // Get All Owned Numbers and Verified Numbers
+  const incomingNumbers = await client.incomingPhoneNumbers.list();
+  const outgoingCallerIds = await client.outgoingCallerIds.list();
+  function finder(numberObject) {
+    return numberObject.phoneNumber === process.env.CALLER_ID;
+  }
+  if (process.env.CALLER_ID) {
+    if (incomingNumbers.find(finder) || outgoingCallerIds.find(finder)) {
+      status.valid = true;
+      status.description = `Your CallerID is set to ${process.env.CALLER_ID}`;
+    } else {
+      status.description = `Your CallerID is set to ${process.env.CALLER_ID}, but that number is not yet verified.
+You can [verify it via the console](https://www.twilio.com/console/phone-numbers/verified), [CLI or API](https://www.twilio.com/docs/voice/api/outgoing-caller-ids).
+`;
+    }
+  } else {
+    status.description = `Your outgoing caller ID can be set to any Twilio number that you've purchased or any numbers that are verified on your account. `;
+    status.actions = incomingNumbers.map(num => ({
+      title: `Choose Twilio # ${num.friendlyName}`,
+      name: "setCallerId",
+      params: {
+        number: num.phoneNumber
+      }
+    }));
+    status.actions = status.actions.concat(
+      outgoingCallerIds.map(num => ({
+        title: `Choose Verified # ${num.friendlyName}`,
+        name: "setCallerId",
+        params: {
+          number: num.phoneNumber
+        }
+      }))
+    );
+  }
+  return status;
+}
+
+async function getTwiMLApplicationIsWiredUp(context) {
+  const client = context.getTwilioClient();
+  const expectedFn = `https://${context.DOMAIN_NAME}/client-voice-twiml-app`;
```

Comment:
similarly you want to avoid absolute paths. There's a new `context.PATH` that you can use to get the current path and then you should construct the new URL based on that since the template might be nested